app/src/main/res/layout/fragment_shift_details.xml

```diff
@@ -0,0 +1,248 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this how all of the page templates for the app are laid out? Each screen is a separate XML file which pulls data from the models?